src/bidiTab/bidiTab.ts

```diff
@@ -55,26 +53,17 @@ const _waitSelfTargetIdPromise = _waitSelfTargetId();
 (async () => {
   MapperTabPage.generatePage();
 
-  const cdpConnection = _createCdpConnection();
-  const bidiServer = _createBidiServer();
-  const eventManager = new EventManager(bidiServer);
-
   // Needed to filter out info related to BiDi target.
   const selfTargetId = await _waitSelfTargetIdPromise;
 
-  // The command processor needs to start running before calling _prepareCdp
-  // so that it has a chance to set up event listeners for tracking targets.
-  await CommandProcessor.run(
-    cdpConnection,
-    bidiServer,
-    eventManager,
-    selfTargetId
-  );
+  const bidiServer = _createBidiServer(selfTargetId);
+
+  await bidiServer.start();
```

Comment:
As mentioned, consider joining `new BidiServer(...)` and `async bidiServer.start()` into `async static BidiServer.start(...)` to ease reading.

Reply:
@sadym-chromium we will likely need to start/stop the instance separately from the creation in Puppeteer so I would keep it separate.

Reply:
`async static BidiServer.create(...)`, as you never need to stop-start the same instance.